### PR TITLE
chore(launch): hide launch internal project from logs 

### PR DESCRIPTION
**Pull request by gtarpenning**
We don't want to show users the `model-registry` project in logs, this scrubs that.

### CODE REVIEW

File: wandb/sdk/internal/internal_api.py [hunk 1]
```diff
@@ -1029,10 +1029,19 @@ def get_project_run_queues(self, entity: str, project: str) -> List[Dict[str, st
...
+                    f"Error fetching run queues for {entity} "
+                    "check that you have access to this entity and project"
+                )
+            else:
+                msg = (
+                    f"Error fetching run queues for {entity}/{project} "
+                    "check that you have access to this entity and project"
+                )
+
+            raise Exception(msg)
```
Comment (kptkin) [inline wandb/sdk/internal/internal_api.py:1044]: why are you raising a general exception here?

File: wandb/sdk/internal/internal_api.py [hunk 2]
```diff
@@ -1029,10 +1029,19 @@ def get_project_run_queues(self, entity: str, project: str) -> List[Dict[str, st
...
+            # circular dependency: (LAUNCH_DEFAULT_PROJECT = model-registry)
+            if project == "model-registry":
+                msg = (
+                    f"Error fetching run queues for {entity} "
+                    "check that you have access to this entity and project"
+                )
+            else:
+                msg = (
+                    f"Error fetching run queues for {entity}/{project} "
+                    "check that you have access to this entity and project"
```
Comment (kptkin) [inline wandb/sdk/internal/internal_api.py:1041]: shouldn't the backend return a permissions errors?

File: wandb/sdk/internal/internal_api.py (hunk 1 above)
Comment (kptkin) [reply wandb/sdk/internal/internal_api.py:1044]: oh i see you kept the old functionality

Comment (gtarpenning) [reply wandb/sdk/internal/internal_api.py:1044]: Yep, I can update it to be more specific if you want :)

File: tests/pytest_tests/system_tests/tests_launch/test_launch.py [hunk 3]
```diff
@@ -128,3 +129,12 @@ def test_launch_multi_run_context(
 
         assert run1.id == "test"
         assert run2.id != "test"
+
+
+def test_launch_get_project_queue_error(user):
+    proj = "projectq32e"
+    api = InternalApi()
+    with pytest.raises(CommError) as e:
```
Comment (kptkin) [inline tests/pytest_tests/system_tests/tests_launch/test_launch.py:137]: you can use the `match` argument in `pytest.raises`. It replaces the assertion you have in the end of the test.

**Review by KyleGoyette — APPROVED** [review]

**Review by kptkin — APPROVED** [review]

**Pull request merged**